src/@next/index.ts

```diff
@@ -36,6 +36,7 @@ export {
 } from './IndexTable';
 export { Pagination, SimplePagination, PaginationProps } from './Pagination';
 export { Popover, PopoverProps } from './Popover';
+export { Radio } from './Radio';
```

Comment:
can we also export the component props here?

Reply:
Can

Reply:
Updated